tests/foreman/ui_airgun/test_settings.py

```diff
@@ -0,0 +1,681 @@
+    """
+    property_name = 'dynflow_enable_console'
+    set_original_property_value(property_name)
+    with session:
+        for param_value in valid_boolean_values():
+            session.settings.update(
+                'name = {}'.format(property_name),
+                param_value
+            )
+            result = session.settings.read('name = {}'.format(property_name))
+            assert result['table'][0]['Value'] == param_value
+
+
+@stubbed()
+@tier1
+def test_negative_settings_access_to_non_admin():
+    """Check non admin users can't access Administer -> Settings tab
+
+    :id: 34bb9376-c5fe-431a-ac0d-ef030c0ab50e
+
+    :steps:
+        1. Login with non admin user
+        2. Check "Administer" tab is not present
+        3. Navigate to /settings
+        4. Check message permission denied is present
+
+    :expectedresults: Administer -> Settings tab should not be available to
+        non admin users
+
+    :CaseImportance: Critical
+    """
+
+
+@stubbed()
+@tier3
+def test_positive_update_email_delivery_method_smtp():
+    """Updating SMTP params on Email tab
+
+    :id: aa4fb26d-2b3a-4b50-891b-a5dfffb31964
+
+    :steps:
+        1. Navigate to Administer > Settings > Email tab
+        2. Update delivery method select interface to SMTP
+        3. SMTP params configuration:
+            3.1. SMTP address
+            3.2. SMTP authentication
+            3.3. SMTP HELO/EHLO domain
+            3.4. SMTP StartTLS  AUTO
+            3.5. SMTP OpenSSL verify mode
+            3.6. SMTP password
+            3.7. SMTP port
+            3.8. SMTP username
+        4. Update "Email reply address" and "Email subject prefix"
+        5. Click "Test Email" button
+        6. Check success msg "Email was sent successfully" is shown
+        7. Check sent email has updated values on sender and subject
+            accordingly
+
+    :expectedresults: Email is sent through SMTP
+
+    :CaseImportance: Critical
+
+    :CaseLevel: Acceptance
+    """
+
+
+@stubbed()
+@tier3
+@upgrade
+def test_negative_update_email_delivery_method_smtp():
+    """Updating SMTP params on Email tab fail
+
+    :id: ecb07f03-f132-4319-b63a-cc3f1ac2c6bb
+
+    :steps:
+        1. Navigate to Administer > Settings > Email tab
+        2. Update delivery method select interface to SMTP
+        3. Update SMTP params with invalid configuration:
+            3.1. SMTP address
+            3.2. SMTP authentication
+            3.3. SMTP HELO/EHLO domain
+            3.4. SMTP password
+            3.5. SMTP port
+            3.5. SMTP port
+            3.6. SMTP username
+        4. Click "Test Email" button
+        5. Check error msg "Unable to send email, check server log for more
+            information" is shown
+        6. Check /var/log/foreman/production.log has error msg related
+            to email
+
+    :expectedresults: Email is not sent through SMTP
+
+    :CaseImportance: Critical
+
+    :CaseLevel: Acceptance
```

Comment:
Please add :CaseAutomation: notautomated for tests which is not automated. May be earlier it was not there, but we should have it now.